lib/kitchen/provisioner/chef_base.rb

```diff
@@ -49,6 +49,7 @@ class ChefBase < Base
       default_config :attributes, {}
       default_config :config_path, nil
       default_config :log_file, nil
+      default_config :log_level, 'auto'
```

Comment:
Is it working with log level as a string? I see the previous defaults were symbols(`:info`, `:auto`)